packages/select/docs/FilterMultiSelect.stories.tsx

```diff
@@ -69,6 +70,63 @@ export const DefaultKaizenSiteDemo = args => {
 
 DefaultKaizenSiteDemo.storyName = "Default (Kaizen Site Demo)"
 
+export const TruncatedLabels = args => {
+  const [selectedKeys, setSelectedKeys] = useState<Selection>(
+    new Set(["id-fe"])
+  )
+  const [characterLimit, setCharacterLimit] = useState<number>(100)
+
+  const handleSelectionChange = (keys: Selection) => {
+    keys && setSelectedKeys(keys)
+  }
```

Comment:
```suggestion
  const handleSelectionChange = (keys: Selection) => setSelectedKeys(keys)
```

Based on your params, `keys` will always be defined